internal/resource/configvalidators/validator_utils.go

```diff
@@ -0,0 +1,117 @@
+package configvalidators
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	internaltypes "github.com/pingidentity/terraform-provider-pingfederate/internal/types"
+)
+
+func IsUrlFormat(aV attr.Value, validateConfigResp *resource.ValidateConfigResponse) diag.Diagnostics {
```

Comment:
Do these need to be in a utils file? They don't seem like they need to be in a shared file rather than just in the corresponding validator file

Reply:
removed the utils file as it's not needed